plugins/services/src/js/utils/__tests__/TaskTableUtil-test.js

```diff
@@ -9,13 +9,14 @@ describe('TaskTableUtil', function () {
   beforeEach(function () {
     this.getServiceHealth = MarathonStore.getServiceHealth;
     MarathonStore.getServiceHealth = function (prop) {
-      return this[prop].get('health');
+      return this[prop]['health'];
```

Comment:
@Poltergeist my main reason to remove the structs is that in this case is only creating more complexity in the code base because the get method is only a abstraction of the native notation which objects are being extended to use the method.

I'm totally fine changing to use the struct but I believe we want to move out using it.